content/60.methods.md

```diff
@@ -271,6 +281,14 @@ These responses were visualized using wordcloud2 in R (<https://CRAN.R-project.o
 Common responses included terms like contribute (44 uses), biology (42 uses), data (38 uses), help (36 uses), learn (32 uses), experience (31 uses), and student (25 uses).
 ](images/interests.png){#fig:wordcloud secno=1}
 
+While some of the challenges we encountered could be remedied by adding functionalities to Manubot, others are more general challenges of Massively Open Online Papers.
+For example, while we automated the updating of data and statistics, the text itself required updating by human experts.
+Given that engagement waxed and waned in this project, as is typical for these types of undertakings [@doi:10.5334/kula.63], this presented challenges at times.
+Some authors returned to update their text, while others did not.
+As a result, the lead authors of each paper often spent several weeks prior to submission updating the text to reflect new developments in each area.
+In the future, it may be possible to streamline this process through integration with a tool such as CoronaCentral [@doi:10.1073/pnas.2100766118] to automatically identify relevant, high-impact papers that need to be included, although expertise would still be required to incorporate them.
+Such a functionality would serve to address another major limitation to this project: ultimately, the scale of the infodemic made it incredibly challenging to stay on top of the COVID-19 literature, even with engagement from a large number of collaborators from diverse backgrounds.
+
```

Comment:
Let's consider additional limitations of the Manubot approach.  Ryan brought up a good one at https://github.com/greenelab/covid19-review/pull/947#pullrequestreview-649162037  Readers of the HTML manuscript cannot tell what parts of the paper are up to date and which are stale.  A savvy reader could use GitHub's blame feature to determine when each sentence or figure was last edited, but that information is not displayed in the rendered manuscript and doesn't distinguish between typo corrections and substantive updates.

A related idea is that we have automating figures and statistics that are summarized by static contributed text.  There is a risk that the data in the figure or statistics can change and the related text will be wrong or inconsistent.  We sometimes tried to work around this by writing "As of date XYZ" in the text, but the core limitation remains.

The pull request model can sometimes be tedious.  Adding even a single sentence requires making a pull request, waiting for an approval, and waiting for the CI build to finish.  That has deterred some colleagues from joining Manubot projects in the past.

There is no good way to provide general comments on a manuscript.  Comments can be added to pull requests, but only to the modified content.  Hypothesis comments can be added anywhere but are not easy to integrate back into the manuscript like pull request comments or suggestions.

Any other limitations we've faced?

Reply:
Another challenge is definitely that we haven't pivoted completely away from DOCX. I have been manually adding commits that people send me either in emails or in docx attachments (using --author). 

Also, there are silly issues like Casey's setting automatically remove EOL whitespace, so every time he opens a PR, it has a huge impact in terms of line numbers. We can fix this by automating this for every PR, but I haven't gotten to it!

Another one-- we don't have a good way to link up preprints with their published papers. I sometimes do this randomly while checking the text -- I'll open a preprint to confirm that I'm not changing the meaning with some edit, and then find that it has been published. I think we can also add this thanks to work David Nicholson has been doing.

Reply:
Other limitations from the reviews we could mention:
- We integrated with the Mount Sinai preprint reviews but any other relevant efforts (e.g. preLights).  There are other potential contributors who are/were reviewing COVID-19 preprints.  How could we build upon their existing efforts?
- scite slows down HTML load times (https://github.com/greenelab/covid19-review/issues/1015).  Could we load the badges only on request?
- Formal user study of Manubot versus other collaborative writing approaches, provide evidence as to how usable (or not) Manubot is for non-technical users.  Bias among current authors toward technical users (related to the point above about DOCX).
- Machine readable/actionable summaries of key information in the review manuscripts as a way to combat the infodemic (as opposed to creating yet another set of manuscripts to read)

Reply:
I believe these are all addressed in the new draft text!